telegram/ext/_precheckoutqueryhandler.py

```diff
@@ -48,6 +48,8 @@ async def callback(update: Update, context: CallbackContext)
         callback (:term:`coroutine function`): The callback function for this handler.
         block (:obj:`bool`): Determines whether the callback will run in a blocking way..
 
+    .. seealso:: `paymentbot.py <https://github.com/python-telegram-bot/
+        python-telegram-bot/blob/master/examples/paymentbot.py>`_
```

Comment:
move above Args and also:
```suggestion
    .. seealso:: `paymentbot.py <examples.paymentbot.html>`_
```